z_te_sts/test_runner.py

```
@@ -0,0 +1,101 @@
+
+        f = io.StringIO()
+        with redirect_stdout(f):
+            spec.loader.exec_module(module)
+
+        output = f.getvalue()
+        if output:
+            print("Output:")
+            print(output)
+
+        print(f"✓ {file_path.name} completed successfully")
+        return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)